### PR TITLE
Fix some tests

### PR DESCRIPTION
**Pull request by jnbrunet**
It should now (after this PR) remains only 5 or 6 failed tests to solve.

Could we reactivate the CI? This would help me tackle the last failed tests.

### CODE REVIEW

File: bindings/Sofa/tests/Core/BaseLink.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def test_read(self):
         root = Sofa.Core.Node("root")
         c1 = root.addObject("MechanicalObject", name="t1")
         link = root.findLink("object")
-        self.assertEqual(link.getValueString(),"@/t1")
+        self.assertEqual(link.getValueString(),"@//t1", F"Link value should be '@//t1', but it is {link.getValueString()}")
```
Comment (damienmarchal) [inline bindings/Sofa/tests/Core/BaseLink.py]: Really ?

File: bindings/Sofa/tests/Simulation/Node.py [hunk 2]
```diff
@@ -141,8 +141,6 @@ def test_getItem(self):
             object2 = node2.addObject("MechanicalObject", name="object2")
 
             # All those are allowed syntaxes:
-            self.assertEqual(root[""].name, root.name)
```
Comment (damienmarchal) [inline bindings/Sofa/tests/Simulation/Node.py]: Why did you remove these ?

**Comment by damienmarchal** [comment]
Hello @jnbrunet 

Thanks you very much for this PR. I fully agree we need a CI but we really needs more manpower :)

File: bindings/Sofa/tests/Simulation/Node.py (hunk 2 above)
Comment (jnbrunet) [reply bindings/Sofa/tests/Simulation/Node.py]: Because when I look at the `__getitem__` binding functions, an empty path (`[""]`) is only supported with objects, and not with nodes:

```
py::object __getitem__(Node& self, const std::string& s) {
(...)
    if(stringlist.empty())
        throw py::value_error("Invalid path provided");
(...)
}
```

Object binding do not have this exception.

File: bindings/Sofa/tests/Core/BaseLink.py (hunk 1 above)
Comment (jnbrunet) [reply bindings/Sofa/tests/Core/BaseLink.py]: This does not seems to come from SofaPython3 side. `"@//t1"` is the value returned by Sofa...

**Comment by jnbrunet** [comment]
Should I merge this?

**Comment by marques-bruno** [comment]
The CI restored would be great yeah... :/
Meanwhile I merge this PR
